app/controllers/user_snps_controller.rb

```diff
@@ -4,9 +4,10 @@ def show
 	end
 
   def index
-    @local_genotype = params[:local_genotype].presence
     if params[:snp_name].present?
-      @user_snps = Snp.find_by_name(params[:snp_name]).user_snps.includes(:user)
+      @local_genotype = params[:local_genotype].presence
+      @genotypes = Genotype.with_local_genotype_for(params[:snp_name])
+                           .includes(:user)
```

Comment:
Use 2 (not 21) spaces for indenting an expression in an assignment spanning multiple lines.